technical-reports/color/color-type.md

```diff
@@ -75,20 +75,20 @@ For example, initially color tokens may be defined as such:
 
 ```json
 {
-  "Majestic magenta": {
+  "Hot pink": {
     "$type": "color",
     "$value": {
-      "$hex": "#c44587",
+      "$hex": "#ff00ff",
       "$colorSpace": {
```

Comment:
Here, and below, these should align with the current spec:

```suggestion
      "colorSpace": "srgb",
      "channels": [1, 0, 1],
```

Reply:
See above comment